Example/ViewController.swift

```diff
@@ -31,23 +31,45 @@ class ViewController: TableViewController {
                 Row(text: "Value 1", detailText: "Detail", cellClass: Value1Cell.self),
                 Row(text: "Value 2", detailText: "Detail", cellClass: Value2Cell.self),
                 Row(text: "Subtitle", detailText: "Detail", cellClass: SubtitleCell.self),
-                Row(text: "Button", detailText: "Detail", cellClass: ButtonCell.self)
+                Row(text: "Button", detailText: "Detail", cellClass: ButtonCell.self, selection: {
+                    [unowned self] in
+                    self.showAlert(title: "Row Selection")
+                }),
             ], footer: "This is a section footer."),
             Section(header: "Accessories", rows: [
                 Row(text: "None"),
                 Row(text: "Disclosure Indicator", accessory: .DisclosureIndicator),
-                Row(text: "Detail Disclosure Button", accessory: .DetailDisclosureButton(detailDisclosureButtonSelected)),
+                Row(text: "Detail Disclosure Button", accessory: .DetailDisclosureButton({
+                    [unowned self] in
```

Comment:
We put this line on the same line as the opening brace in our app. I don’t really mind whichever way, but we should be consistent :sparkles: